add shutdown statement (#2865)
pingcap/docs: sql-statement-shutdown.md

> ---

Reviewer: Please add this file to TOC.md @jackysp

title: SHUTDOWN

> summary: Overview of the use of SHUTDOWN in the TiDB database.

Reviewer 2: ```suggestion
summary: An overview of the usage of SHUTDOWN for the TiDB database.
```

category: reference
---

# SHUTDOWN

The `SHUTDOWN` statement is used to perform a shutdown operation in TiDB. Execution of the `SHUTDOWN` statement requires the user to have `SHUTDOWN privilege`.

## Synopsis

**Statement:**

![Statement](/media/sqlgram/ShutdownStmt.png)

## Examples

{{< copyable "sql" >}}

```sql
SHUTDOWN;
```

```
Query OK, 0 rows affected (0.00 sec)
```

## MySQL compatibility

> > **Attention:**

Reviewer 2: ```suggestion
> **Note:**
```

>

> > Since TiDB is a distributed database, the shutdown operation in TiDB stops the client-connected TiDB instance, not the entire TiDB cluster.

Reviewer 2: ```suggestion
> Because TiDB is a distributed database, the shutdown operation in TiDB stops the client-connected TiDB instance, not the entire TiDB cluster.
```

> The `SHUTDOWN` statement is partly compatible with MySQL. Any compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.

Reviewer 2: ```suggestion
The `SHUTDOWN` statement is partly compatible with MySQL. If you encounter any compatibility issues, you can report the issues [on GitHub](/report-issue.md).
```
